antissh.conf.example

```diff
@@ -7,6 +7,8 @@ oper = x x
 #modes = +s +c
 nickname = antissh
 kline_cmd = KLINE 86400 *@{ip} :Vulnerable SSH daemon found on this host.  Please fix your SSH daemon and try again later.
+# Log when vulnerable hosts are found to this channel (no #)
+#log_chan = antissh-log
```

Comment:
Why not explicitly write the channel prefix? That would give better support for local &channels and make ##channels less confusing should someone want to use them.

Reply:
I was worried it would be interpreted as a comment, but after reviewing [the configparser docs](https://docs.python.org/3/library/configparser.html#customizing-parser-behaviour) I see that that is not the case. `inline_comment_prefixes` is None by default.